### DISCUSSION TITLE
Configurable Pillager Raid Quantities

### DISCUSSION THREAD

**Discussion by Munchkinnibbler** [PurpurMC/Purpur#900] [Feature Requests]
Looks like my previous attempt at suggesting this has vanished, apologies if it is buried somewhere and I've been unable to find it, however.

It would be great if there was a config to specify the number of varying raider type without having to change the difficulty of the server as a whole. For example, you could be able to specify the max number of evokers spawned. Whether that is then further specified by raid level is debatable but some way of controlling the number of each raider type spawned would be lovely. I suppose, if this is even possible which it might not be, it would then make an easy way to implement/control illusioners in raids as I saw suggested in #553 .

In case it's helpful, the possible raider types are pillager,  ravager, vindicator, witch, evoker, ravager+pillager and ravager+evoker.

**Comment by alexmcgowan112** [comment]
It could also be cool to include some configuration for the illusioner.
